tsl/test/sql/chunk_utils_internal.sql

```diff
@@ -0,0 +1,91 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+--  These tests work for PG14 or greater
+
+\c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE SCHEMA test1;
+GRANT CREATE ON SCHEMA test1 TO :ROLE_DEFAULT_PERM_USER;
+GRANT USAGE ON SCHEMA test1 TO :ROLE_DEFAULT_PERM_USER;
+
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+CREATE TABLE test1.hyper1 (time bigint, temp float);
+
+SELECT create_hypertable('test1.hyper1', 'time', chunk_time_interval => 10);
+
+INSERT INTO test1.hyper1 VALUES (10, 0.5);
+
+INSERT INTO test1.hyper1 VALUES (30, 0.5);
+SELECT chunk_schema as "CHSCHEMA",  chunk_name as "CHNAME", 
+       range_start_integer, range_end_integer
+FROM timescaledb_information.chunks
+WHERE hypertable_name = 'hyper1' and hypertable_schema = 'test1'
+ORDER BY chunk_name ;
+
+--TEST internal api that freezes a chunk
+--freeze one of the chunks
+SELECT chunk_schema || '.' ||  chunk_name as "CHNAME"
+FROM timescaledb_information.chunks
+WHERE hypertable_name = 'hyper1' and hypertable_schema = 'test1'
+ORDER BY chunk_name LIMIT 1
+\gset
+
+SELECT  _timescaledb_internal.freeze_chunk( :'CHNAME');
```

Comment:
Also suggesting to add a similar case for a distributed chunk, just to see that it properly fails

Reply:
done.